tiny_gltf.h

```diff
@@ -1704,7 +1705,7 @@ static bool ParseExtensionsProperty(ExtensionMap *ret, std::string* err, const j
     ParameterMap extValues;
     for (; itVal != itValEnd; itVal++) {
       Parameter param;
-      if (ParseParameterProperty(&param, err, values_object, itVal.key(), false)) {
+      if (ParseParameterProperty(&param, err, itVal.value(), itVal.key(), false)) {
```

Comment:
I have just found that this change actually broke the extension parsing.

I will be changing the ExtensionMap to use Value objects instead, so this'll be fixed along the way.